README.md

```diff
@@ -1,4 +1,23 @@
-# OpenCensus-Exporter
-Datadog exporter for OpenCensus metrics
-
+# Datadog OpenCensus Metrics Exporter
 Owned by Team Agent-Integrations (Tiramisu)
```

Comment:
If we're putting this into the [census-ecosystem](https://github.com/census-ecosystem) will this make sense?

Reply:
Good catch, I forgot to ask but we should remove that reference